[2025-09-02T21:16:07.604+0800] INFO - local_task_job_runner.py:123 - _execute() - ::group::Pre task execution logs
[2025-09-02T21:16:07.830+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_dummy_tasks_dag.start scheduled__2025-08-31T18:00:00+00:00 [queued]>
[2025-09-02T21:16:07.907+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_dummy_tasks_dag.start scheduled__2025-08-31T18:00:00+00:00 [queued]>
[2025-09-02T21:16:07.908+0800] INFO - taskinstance.py:2866 - _check_and_change_state_before_execution() - Starting attempt 1 of 2
[2025-09-02T21:16:08.364+0800] INFO - taskinstance.py:2889 - _check_and_change_state_before_execution() - Executing <Task(PythonOperator): start> on 2025-08-31 18:00:00+00:00
[2025-09-02T21:16:08.416+0800] INFO - standard_task_runner.py:72 - _start_by_fork() - Started process 212 to run task
[2025-09-02T21:16:08.539+0800] INFO - standard_task_runner.py:104 - _start_by_fork() - Running: ['***', 'tasks', 'run', 'example_dummy_tasks_dag', 'start', 'scheduled__2025-08-31T18:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/example_dummy_tasks_dag.py', '--cfg-path', '/tmp/tmpqgylv6o9']
[2025-09-02T21:16:08.546+0800] INFO - standard_task_runner.py:105 - _start_by_fork() - Job 2: Subtask start
[2025-09-02T21:16:09.053+0800] INFO - task_command.py:467 - task_run() - Running <TaskInstance: example_dummy_tasks_dag.start scheduled__2025-08-31T18:00:00+00:00 [running]> on host a246a11defc8
[2025-09-02T21:16:09.915+0800] INFO - taskinstance.py:3132 - _execute_task_with_callbacks() - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='example_dummy_tasks_dag' AIRFLOW_CTX_TASK_ID='start' AIRFLOW_CTX_EXECUTION_DATE='2025-08-31T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-31T18:00:00+00:00'
[2025-09-02T21:16:09.931+0800] INFO - taskinstance.py:731 - _execute_callable() - ::endgroup::
[2025-09-02T21:16:10.235+0800] INFO - logging_mixin.py:190 - _propagate_log() - Hello from Airflow!
[2025-09-02T21:16:10.236+0800] INFO - python.py:240 - execute() - Done. Returned value was: None
[2025-09-02T21:16:10.348+0800] INFO - taskinstance.py:340 - _run_raw_task() - ::group::Post task execution logs
[2025-09-02T21:16:10.349+0800] INFO - taskinstance.py:352 - _run_raw_task() - Marking task as SUCCESS. dag_id=example_dummy_tasks_dag, task_id=start, run_id=scheduled__2025-08-31T18:00:00+00:00, execution_date=20250831T180000, start_date=20250902T131607, end_date=20250902T131610
[2025-09-02T21:16:10.517+0800] INFO - local_task_job_runner.py:266 - handle_task_exit() - Task exited with return code 0
[2025-09-02T21:16:10.622+0800] INFO - taskinstance.py:3895 - _schedule_downstream_tasks() - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-02T21:16:10.638+0800] INFO - local_task_job_runner.py:245 - _execute() - ::endgroup::
